docs/cli.md

```diff
@@ -1101,20 +1104,23 @@ viam machines part restart --machine=<machine id> --part=<part id>
 Examples:
 
 ```sh {class="command-line" data-prompt="$"}
-
 # list all machines you have access to
 viam machines list
 
 # get machine status
 viam machines status  --machine 82c608a-1be9-46a5-968d-bad3a8a6daa --organization "Robot's Org" --location myLoc
 
-# stream error level logs from a machine part
+# stream logs from a machine
 viam machines part logs --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
---organization "Robot's Org" --location myLoc --part "myrover-main" --tail true
+--organization "Robot's Org" --location myLoc
+
+# stream logs from a machine part
+viam machines part logs --machine 82c608a-1be9-46a5-968d-bad3a8a6daa \
```

Comment:
is this a real machine value?

Reply:
Not sure but it looks like it so I will change to 123